### ISSUE TITLE
Add Project Search option.

### ISSUE THREAD

**Issue by ZoomyCat** [benjaminRomano/todo-manager#15]
I'd like to be able to see all the TODO statements within the project to be able to see only the relevent information for what I'm currently working on.

I find the extra data of `All Files` distracting because one of the tools I currently use generates some code coverage. This tool appears to have some TODO statement in the code generation and negatively affects my ability to use my development TODO listing.

Otherwise absolutely fantastic tool. I love it!

**Comment by benjaminRomano** [comment]
I agree it would be nice to be able to ignore those auto-generated TODOs. A solution to this problem would be to add a ignoreFiles field to the todo-manager config and preferably allow using glob to pattern match files.

I don't have time atm to make the changes. If you have some time and are interested in doing it, please feel free to make a PR.
